### PR TITLE
Calicoctl docs

### PR DESCRIPTION
**Pull request by robbrockbank**

### CODE REVIEW

File: docs/calicoctl/bgp.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+This sections describes the `calicoctl bgp` commands.
+
+These commands can be used to manage the global BGP configuration.  This 
+includes default values to use for the AS number, whether a full BGP mesh is
+required between all of the Calico nodes, and global BGP peers (these are BGP
+speakers that peer with every Calico node in the network).
+
+In addition to global BGP configuration, there is also Calico node specific
+BGp configuration.  This provides mechanisms to set up BGP peers for an
```
Comment (alexwlchan) [inline docs/calicoctl/bgp.md]: “BGp” ~> “BGP”

File: docs/calicoctl/bgp.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+$ calicoctl bgp peer show --ipv4
++----------------------+--------+
+| Global IPv4 BGP Peer | AS Num |
++----------------------+--------+
+| 192.0.2.10           | 64555  |
++----------------------+--------+ 
+```
+
+### calicoctl bgp node-mesh 
+This command is used view the status of, or enable and disable, the full
```
Comment (alexwlchan) [inline docs/calicoctl/bgp.md]: Should this be “used to view”?

File: docs/calicoctl/bgp.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+$ calicoctl bgp node-mesh
+off
+```
+
+### calicoctl bgp default
+This command is used to view and set the default AS number used by Calico 
+nodes.
+
+When a Calico node is started (see [calicoctl node](node.md) commands)
+the default AS number is used when confiburing the BGP peerings if one has not
```
Comment (alexwlchan) [inline docs/calicoctl/bgp.md]: “confiburing” ~> “configuring”

File: docs/calicoctl/container.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+Examples:
+
+```
+$ calicoctl container test-container ip add 192.10.0.3 --interface=eth1 
+```
+
+### calicoctl container <CONTAINER> ip remove <IP> 
+
+This command allows you t remove an IP address from a container that is
```
Comment (alexwlchan) [inline docs/calicoctl/container.md]: “t remove” ~> “to remove”

**Comment by robbrockbank** [comment]
This is very much a work in progress - just for high level feedback on format and level of detail (on the ones that I've padded out).

**Comment by tomdee** [comment]
@robbrockbank Let's discuss when you're back in the office.

File: release_process/update_docs.py [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+if __name__ == '__main__':
+    """
+    Parse the calicoctl doc strings and update the calicoctl documentation
+    with the latest help text.
+
+    This should be done after making any updates to the calicoctl commands,
+    and as part of the release process.
+
+    If this applies any changes to the documentation files it is necessary to
+    check the updated text byu hand and apply any manual updates as necessary.
```
Comment (TrimBiggs) [inline release_process/update_docs.py]: byu -> by

**Comment by tomdee** [comment]
@TrimBiggs Am I right in thinking that you're taking this on?

**Comment by TrimBiggs** [comment]
@tomdee That's right.

**Comment by tomdee** [comment]
@TrimBiggs @robbrockbank Can this be closed?

**Comment by robbrockbank** [comment]
@tomdee - yes.

Closing - PR is now covered by #532

**Pull request closed without merging**